biolink-model.yaml

```diff
@@ -6549,6 +6555,170 @@ slots:
       - NCIT:C44185
       - EDAM-DATA:1669
 
+  concept count subject:
+    is_a: association slot
+    description: >-
+      The number of instances in a dataset/cohort whose records contain
+      the concept in the subject slot of an association.
+    range: integer
+    examples:
+      - value: 489
+
+  concept count object:
+    is_a: association slot
+    description: >-
+      The number of instances in a dataset/cohort whose records contain
+      the concept in the object slot of an association.
+    range: integer
+
+  concept pair count:
+    is_a: association slot
+    description: >-
+      The number of instances in a dataset/cohort whose records contain
+      both the subject and object concept of an association.
+    range: integer
+    examples:
+      - value: 1731
+
+  expected count:
+    is_a: association slot
+    description: >-
+      The expected (calculated) number of instances in a dataset/cohort whose records contain both the subject and 
+      object concept of an association if the subject and object concepts are independent.
+
+  relative frequency subject:
+    is_a: association slot
+    description: >-
+      The frequency at which subject and object concepts co-occur in EHR
+      records within a dataset/cohort, relative to the frequency at which the subject
+      concept appears in these same records.
+    range: float
+    examples:
+      - value: 0.01840490798
+
+  relative frequency object:
+    is_a: association slot
+    description: >-
+      The frequency at which subject and object concepts co-occur in EHR
+      records within a dataset/cohort, relative to the frequency at which the object
+      concept appears in these same records.
+    range: string
+
+  relative frequency subject confidence interval:
+    is_a: association slot
+    description: >-
+      The 99% confidence interval for the relative_frequency_subject calculation
+      (i.e. the range of values within which the true value has a 99% chance of falling)
+    range: string
+
+  relative frequency object confidence interval:
+    is_a: association slot
+    description: >-
+      The 99% confidence interval for the relative_frequency_object calculation
+      (i.e. the range of values within which the true value has a 99% chance of falling)
+    range: string
+
+  adjusted p value:
+    is_a: association slot
```

Comment:
Seems like we should have a hierarchy of more specific p-value flavors.  a generic root 'p-value', with child slots for 'adjusted p-value', 'unadjusted p-value', 'bonferoni adjusted p-value' (which may actually go under 'adjusted p-value') . . . thoughts?  Maybe keep flat for now, and save this for next time?